docs/validation/guides/validate-ODF-LSO.md

```diff
@@ -0,0 +1,53 @@
+# Validate Deployment of OpenShift Data Foundation (ODF) with LSO
+
+The steps below describe how to run acceptance tests manually for ODF (OpenShift Data Foundation) using LSO (Local Storage Operator).
+
+Prerequisites
+
+- OpenShift Cluster Installed in version 4.13 (or newer). Export the variable `CLUSTER_VERSION`. Example: `export CLUSTER_VERSION=4.13`
+- Cluster name exported in the variable `CLUSTER_NAME`
+- Cluster administrator credentials is saved under ${INSTALL_DIR}/auth, example:
+
+~~~bash
+$ tree ${INSTALL_DIR}
+└── auth
+    ├── kubeadmin-password
+    └── kubeconfig
+~~~
+
+Steps:
+
+1) Pull the image:
+
+```bash
+ podman login quay.io
+ podman pull quay.io/ocsci/ocs-ci-container:stable
+```
+
+2) Run a simple test:
+
+> The ":Z" option for the volume mount can be omited when running in OS different than Linux.
+
+~~~bash
+podman run -v ${INSTALL_DIR}:/opt/cluster:Z quay.io/ocsci/ocs-ci-container:stable run-ci --cluster-path /opt/cluster --ocp-version ${CLUSTER_VERSION} --ocs-version ${CLUSTER_VERSION} --cluster-name ${CLUSTER_NAME} tests/manage/z_cluster/test_noobaa_xss_vulnerability.py --dev-mode
+~~~
+
+3) Run acceptance suite:
+
+~~~bash
+podman run -v ${INSTALL_DIR}:/opt/cluster:Z quay.io/ocsci/ocs-ci-container:stable run-ci -m acceptance --junit-xml /opt/cluster/junit.xml --cluster-path /opt/cluster --ocp-version ${CLUSTER_VERSION} --ocs-version ${CLUSTER_VERSION} --color=yes --cluster-name ${CLUSTER_NAME} tests/
+~~~
+
+> The results must be saved in `${INSTALL_DIR}/junit.xml`.
+
+4) Collect the Must Gather for ODF
+
+~~~bash
+oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9
+~~~
```

Comment:
MG cmd failed with this image ```$ oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9
[must-gather      ] OUT Using must-gather plug-in image: registry.redhat.io/odf4/odf-must-gather-rhel9
When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
ClusterID: 90e29e2c-b6bc-42e8-b980-dc7bad925878
ClusterVersion: Stable at "4.13.0-0.nightly-2023-06-19-110814"
ClusterOperators:
	clusteroperator/machine-config is degraded because Failed to resync 4.13.0-0.nightly-2023-06-19-110814 because: error during syncRequiredMachineConfigPools: [timed out waiting for the condition, error pool worker is not ready, retrying. Status: (pool degraded: true total: 3, ready 2, updated: 2, unavailable: 0)]


[must-gather      ] OUT namespace/openshift-must-gather-nzxg8 created
[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-2m9zg created
[must-gather      ] OUT pod for plug-in image registry.redhat.io/odf4/odf-must-gather-rhel9 created
[must-gather-9nf66] OUT gather did not start: unable to pull image: ImagePullBackOff: Back-off pulling image "registry.redhat.io/odf4/odf-must-gather-rhel9"
```

Reply:
thanks, idk where that image came from :)
I just added the available in the docs https://docs.openshift.com/container-platform/4.10/support/gathering-cluster-data.html